File: spec/code_samples/Python_requests/end_user_authentications@passwords/patch.py
import requests

url = "https://www.directfreight.com/api/end_user_authentications/passwords"

payload = "{\"new_password\":\"<ADD STRING VALUE>\",\"old_password\":\"<ADD STRING VALUE>\"}"
response = requests.request("PATCH", url, data=payload)

print(response.text)

File: spec/code_samples/Php_http2/end_user_authentications@passwords/patch.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$body = new http\Message\Body;
$body->append('{"new_password":"<ADD STRING VALUE>","old_password":"<ADD STRING VALUE>"}');

$request->setRequestUrl('https://www.directfreight.com/api/end_user_authentications/passwords');
$request->setRequestMethod('PATCH');
$request->setBody($body);

$request->setHeaders(array(
  'end-user-token' => 'SOME_STRING_VALUE'
));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();